
#########################  点云筛选  #######################################
use_img_color : false

cam_topic : /usb_cam/image_raw

lidar_topic : /livox/lidar
#lidar_topic : /velodyne_cloud_registered

#没有imu里程计数据时为false
use_tf_inform : false

#相机与雷达标定结果
tc_l : [ 0.0817661,-0.996518,0.0162978,-0.0548113,
0.091371,-0.00878855,-0.995778,   -0.0842716,
0.992454,0.0829101,0.0903343,    0.163149,
0,0,0,1]

#相机内参
camera_internal_matrix: [  879.561224,        0.0,      1000.720510,
                               0.0,    879.727041,      549.385583,
                               0.0,        0.0,          1.0 ]


#################################  最高点检测  ##############################

check_cloud : /livox/lidar

# 如果使用以下参数，表示使用筛选后的点云进行最高点检测
#check_cloud : /roi_point_cloud

# 轨条最高点至少离“地面”的距离
min_distance : 0.5

# 轨条最高点至少离雷达的距离，用于过滤噪声
min_lidar_distance : 0.05

# 发布一次数据时融合的帧数
frame_tick : 10

# 统计滤波邻近点数
filter_neighbor_points : 10

# 统计滤波离群点阈值
filter_threshold : 2.0

# 开启测试模式，会输出便于调试的中间结果，但是会额外占用性能
debug_mode : false

